parquet-variant/src/builder.rs

```diff
@@ -70,6 +70,13 @@ fn write_offset_at_pos(buf: &mut [u8], start_pos: usize, value: usize, nbytes: u
     buf[start_pos..start_pos + nbytes as usize].copy_from_slice(&bytes[..nbytes as usize]);
 }
 
+/// Append `value_bytes` of given `value` into `dest`.
```

Comment:
`value_bytes` is the byte width of the value?
```suggestion
/// Append `value_bytes` bytes of given `value` into `dest`.
```

Reply:
fixed